app/src/main/java/com/kylecorry/trail_sense/navigation/ui/LinearCompassView.kt

```diff
@@ -150,7 +150,7 @@ class LinearCompassView : CanvasView, ICompassView {
                             else -> ""
                         }
                         noStroke()
-                        fill(Resources.androidTextColorPrimary(context))
+                        fill(Resources.color(context, R.color.colorPrimary))
```

Comment:
I tried this out, and am not a fan of how it looks - I would prefer this text to remain text color primary (and the direction at the top of the screen can be used on the non-contrasting backgrounds).